CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.3.1]
```

Comment:
I wouldn't bother with this commit: we don't try to keep the changelog up-to-date on branches other than master. It's too much work (e.g. v1.3 doesn't have a release date).

Be sure to open a separate PR to add 1.3.1 to master, however. We should add more context that this is a special build: it has no commit differences but is a rebuild of v1.3 and why we need that (formatted like the additional context for v1.1 with italics and such).

fwiw, we should always keep the `## [Unreleased]` header.